app/controllers/members_controller.rb

```diff
@@ -38,14 +38,29 @@ def index
   end
 
   def create
-    service_call = create_members
+    overall_result = []
```

Comment:
Here you are initializing with an empty array, [in the other controller with `nil`](https://github.com/opf/openproject/pull/14033/files#diff-1044ba07b8889f8ff65fb7b1338806f8909221c74d285aadcb659334a3ca169dR43). 

When we use the empty array, we don't need the conditional `if overall_result` in line 48 here and can push directly into the array. I think this makes more sense and we should use that approach in both controllers as it needs one less if conditional

Reply:
Good catch! Thanks 👍